langchain/src/vectorstores/vectara.ts

```diff
@@ -1,3 +1,8 @@
+import fs from "fs";
+import path from "path";
```

Comment:
This should be built into Node no?

Reply:
Yes. I just removed the path lib from the package.json